tc3-japan/website-crawler
# add content and category, update solr search logic and highlight

How to verify ?

1.  try to remove all tables, then run `./gradlew flywayMigrate` to create tables and insert data

2.  `./gradlew bootRun -Pargs=--site=1,--proc=crawler` to fetch data, then check content, category in pages table

3.  `./gradlew bootRun -Pargs=--site=1 --proc=converter` to convert data into solr 

4. try to invoke POST /api/search_products with body  like this 

```
{
	"query":["WOMEN"],
	"start":0,
	"rows":10,
	"firstNOfContent":"3000"
}
``` to  check response
